src/TodoList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TodoList = ({ todos, destroy, checked, checkedAll }) => (
+
+  <section className="main" style={{ display: 'block' }}>
+    <label htmlFor="toggle-all">
+      <input
+        type="checkbox"
+        id="toggle-all"
+        className="toggle-all"
+        checked={todos.every(i => i.completed) && todos.length !== 0}
+        onChange={(event) => {
+          checkedAll(event.target.checked);
+        }
+        }
+      />
+    </label>
+
+    <ul className="todo-list">
+      {todos.map(item => (
+        <li key={item.id} className="">
```

Comment:
```suggestion
        <li key={item.id} >
```
Если тебе не нужен className - не пиши его